tests/ChangeLog

```diff
@@ -1,3 +1,7 @@
+2022-01-27  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
+
+	* testsuite.src/syn_control.at: new testsuite for CONTROL DIVISION
```

Comment:
Let's move this to an existing syn group, if we have support for many `CONTROL DIVISION` entries we can still move it out to a new one.